src/components/comments/index.test.ts

```diff
@@ -25,45 +52,114 @@ describe('CommentsComponent', () => {
   });
 
   it('should have an element property', () => {
-    expect(commentsComponent['element']).toBeUndefined();
+    expect(commentsComponent['element']).toBeDefined();
   });
 
   it('should create a new element when start() is called', () => {
-    commentsComponent.attach({
-      realtime: ABLY_REALTIME_MOCK,
-      localParticipant: MOCK_LOCAL_PARTICIPANT,
-      group: MOCK_GROUP,
-      config: MOCK_CONFIG,
-      eventBus: EVENT_BUS_MOCK,
-    });
     commentsComponent.detach();
     expect(commentsComponent['element']).toBeUndefined();
   });
 
   it('should add the element to the document body when start() is called', () => {
-    commentsComponent.attach({
-      realtime: ABLY_REALTIME_MOCK,
-      localParticipant: MOCK_LOCAL_PARTICIPANT,
-      group: MOCK_GROUP,
-      config: MOCK_CONFIG,
-      eventBus: EVENT_BUS_MOCK,
-    });
-
     expect(document.body.contains(commentsComponent['element'])).toBe(true);
   });
 
   it('should remove the element from the document body when destroy() is called', async () => {
-    commentsComponent.attach({
-      realtime: ABLY_REALTIME_MOCK,
-      localParticipant: MOCK_LOCAL_PARTICIPANT,
-      group: MOCK_GROUP,
-      config: MOCK_CONFIG,
-      eventBus: EVENT_BUS_MOCK,
-    });
     expect(commentsComponent['element']).toBeDefined();
 
     commentsComponent.detach();
 
     expect(document.body.contains(commentsComponent['element'])).toBe(false);
   });
+
+  it('should call apiService when fetch annotation', async () => {
+    const result = jest.spyOn(ApiService, 'fetchAnnotation');
+
+    expect(result).toHaveBeenCalledWith(
+      expect.any(String),
+      expect.any(String),
+      {
+        roomId: expect.any(String),
+        url: expect.any(String),
+      },
+    );
+  });
```

Comment:
Calling this variable result makes it look like it's function return, when it's a function spy. 

```suggestion
    const spy = jest.spyOn(ApiService, 'fetchAnnotation');

    expect(spy).toHaveBeenCalledWith(
      expect.any(String),
      expect.any(String),
      {
        roomId: expect.any(String),
        url: expect.any(String),
      },
    );
  });
```